### PR TITLE
Code of Conduct Updates

### PR DESCRIPTION
**Pull request by adriennefranke**
## Description

This adds examples of unacceptable behavior and procedures for reporting incidents to the Code of Conduct. These were taken from PyCon's Code of Conduct which is under the [Creative Commons Attribution-ShareAlike 3.0 Unported License](https://creativecommons.org/licenses/by-sa/3.0/).

### CODE REVIEW

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -71,6 +71,27 @@ should not use sexualized images, activities, or other material. Booth staff
 (including volunteers) should not use sexualized clothing/uniforms/costumes,
 or otherwise create a sexualized environment.
 
+## Examples of Inappropriate Behavior</h2>
```
Comment (tim-schilling) [inline CODE_OF_CONDUCT.md:74]: There's a trailing `</h2>` here. Is that intended?

File: CODE_OF_CONDUCT.md [hunk 2]
```diff
@@ -71,6 +71,27 @@ should not use sexualized images, activities, or other material. Booth staff
...
+- Excessive swearing
+- Unwelcome sexual attention or advances
+- Unwelcome physical contact, including simulated physical contact (eg, textual descriptions like “hug” or “backrub”) without consent or after a request to stop
+- Pattern of inappropriate social contact, such as requesting/assuming inappropriate levels of intimacy with others
+- Sustained disruption of online community discussions, in-person presentations, or other in-person events
+- Continued one-on-one communication after requests to cease
+- Other conduct that is inappropriate for a professional audience including people of many different backgrounds
+
+Community members asked to stop any inappropriate behavior are expected to comply immediately.
+
```
Comment (tim-schilling) [inline CODE_OF_CONDUCT.md:94]: We should probably include a mention that this list is not an exhaustive list of all inappropriate behavior that is unacceptable at DjangoCon US.

File: _pages/conduct.html [hunk 3]
```diff
@@ -123,6 +154,29 @@ <h2>Choose Your Words</h2>
...
+          <li>Excessive swearing</li>
+          <li>Unwelcome sexual attention or advances</li>
+          <li>Unwelcome physical contact, including simulated physical contact (eg, textual descriptions like “hug” or “backrub”) without consent or after a request to stop</li>
+          <li>Pattern of inappropriate social contact, such as requesting/assuming inappropriate levels of intimacy with others</li>
+          <li>Sustained disruption of online community discussions, in-person presentations, or other in-person events</li>
+          <li>Continued one-on-one communication after requests to cease</li>
+          <li>Other conduct that is inappropriate for a professional audience including people of many different backgrounds</li>
+        </ul>
+        
+        <p>Community members asked to stop any inappropriate behavior are expected to comply immediately.</p>
```
Comment (tim-schilling) [inline _pages/conduct.html:179]: Any change to the markdown file for the non-exhaustive list comment will need to be duplicated here.

File: _pages/conduct.html [hunk 4]
```diff
@@ -136,6 +190,8 @@ <h2>License</h2>
 
         <p>This Code of Conduct was forked from the example policy from the <a href="http://geekfeminism.wikia.com/wiki/Conference_anti-harassment/Policy">Geek Feminism wiki</a>, created by the Ada Initiative and other volunteers. which is under a Creative Commons Zero license.</p>
 
+        <p>This Code of Conduct also incorporates portions from PyCon's Code of Conduct, which is licensed under the <a href="https://creativecommons.org/licenses/by-sa/3.0/">Creative Commons Attribution-ShareAlike 3.0 Unported License</a>.
```
Comment (tim-schilling) [inline _pages/conduct.html:193]: I think this is missing a closing `</p>` tag.

**Review by tim-schilling — CHANGES_REQUESTED** [review]
I like these changes a lot!

**Review by drewbrew — APPROVED** [review]
Aside from Tim's markup/markdown fixes, LGTM

**Review by tim-schilling — APPROVED** [review]
Thanks for making those changes and doing this work!

**Pull request merged**